run_tests.sh

```diff
@@ -1,5 +1,26 @@
 #!/usr/bin/bash
 
+for dockerfile in Dockerfile-reposcan Dockerfile-webapp Dockerfile-websocket
```

Comment:
Dockerfile-database is missing

Reply:
`Dockerfile-database and Dockerfile-database.rhel7 are too different!`

it's because of hardcoded PG 10 value in test, please add a commit switching database to PG 10

also make sure run_tests.sh fails if error is detected